src/main/io/vtx_smartaudio.c

```diff
@@ -1036,7 +1036,7 @@ static uint8_t vtxSAGetPowerLevels(const vtxDevice_t *vtxDevice, uint16_t *level
 
     for (uint8_t i = 0; i < saSupportedNumPowerLevels; i++) {
         levels[i] = saSupportedPowerValues[i];
-        uint16_t power = (uint16_t)pow(10.0,levels[i]/10.0);
+        uint16_t power = (uint16_t)powf(10.0f,levels[i]/10.0f);
```

Comment:
This appears unrelated to USE_GPS_PLUS_CODES

Reply:
Changing `pow` to `powf` is needed because it enables linker to ommit `double pow(double, double)`. This is the only one place where it is used.

Chnaginf `10.0` to `10.0f` is probably not needed because of `-fsingle-precision-constant`.

Reply:
I also did this in #11096 if you want separate PR.

Reply:
Change `10.0f` to `10.0` to make minimal changes. We don't need it due to `-fsingle-precision-constant`